backend/scripts/run_tests.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Script to run all tests in the open_webui/test directory
+
+# Ensure we're in the backend directory
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)